src/helpers/player.js

```diff
@@ -0,0 +1,42 @@
+import { teams, tileTypes } from '../constants';
+
+let me;
+
+// Sets up the "me" player for this helper file.
+export function setupMe() {
+  me = tagpro.players[tagpro.playerId];
+}
+
+// For testing purposes.  Sets me.team to RED
```

Comment:
move all the testing function to the bottom of the file, and comment "all following functions are for testing only"